src/pkg/packager/deploy.go

```diff
@@ -124,10 +133,12 @@ func (p *Packager) Deploy(ctx context.Context) error {
 	}
 	if len(deployedComponents) == 0 {
 		message.Warn("No components were selected for deployment.  Inspect the package to view the available components and select components interactively or by name with \"--components\"")
+		l.Warn("No components were selected for deployment.  Inspect the package to view the available components and select components interactively or by name with \"--components\"")
 	}
 
 	// Notify all the things about the successful deployment
 	message.Successf("Zarf deployment complete")
+	l.Debug("Zarf deployment complete", "duration", time.Since(start))
```

Comment:
Nit: capital. 👍 on the duration metric

Reply:
This one was on purpose. I've been capitalizing proper nouns like Zarf and Helm. I'm fine with either choice, lmk what you'd like to do going forward, as I currently have the word Helm capitalized a few times in this PR.